classes/class-wc-connect-shipping-label.php

```diff
@@ -107,12 +115,12 @@ protected function get_form_data( WC_Order $order ) {
 			$form_data[ 'destination' ] = array(
 				'name' => trim( $dest_address[ 'first_name' ] . ' ' . $dest_address[ 'last_name' ] ),
 				'company' => $dest_address[ 'company' ],
-				'address_1' => $dest_address[ 'address_1' ],
+				'address' => $dest_address[ 'address_1' ],
 				'address_2' => $dest_address[ 'address_2' ],
 				'city' => $dest_address[ 'city' ],
 				'state' => $dest_address[ 'state' ],
 				'postcode' => $dest_address[ 'postcode' ],
-				'country' => $dest_address[ 'country' ],
+				'country' => $dest_address[ 'country' ] ? $dest_address[ 'country' ] : 'US',
```

Comment:
Under what circumstances would country be blank? Only in the case of a manually entered order?